reana-ui/src/pages/profile/components/GitLabProjects.js

```diff
@@ -29,7 +29,11 @@ export default function GitLabProjects() {
       client
         .getGitlabProjects()
         .then((res) => {
-          setProjects(res.data);
+          let data = res.data;
+          for (let projectId in data) {
+            data[projectId].toggling = false;
+          }
```

Comment:
Very minor: instead of mutating the data object we could do it like this:
```
const data = Object.entries(res.data).map(([id, val]) => ({
  [id]: { ...val, toggling: false },
}));
```
but I guess it's a bit less readable, so we can leave it as is

Reply:
```js
let newProjects = {};
for (const [id, details] of Object.entries(res.data)) {
  newProjects[id] = {
    ...details,
    toggling: false,
  };
}
```

What do you think?

Reply:
Looks good :+1: